supervisor/backups/backup.py

```diff
@@ -387,30 +387,27 @@ def _folder_save(name: str):
                 return
 
             # Take backup
-            try:
-                _LOGGER.info("Backing up folder %s", name)
-                with SecureTarFile(
-                    tar_name, "w", key=self._key, gzip=self.compressed
-                ) as tar_file:
-                    atomic_contents_add(
-                        tar_file,
-                        origin_dir,
-                        excludes=[],
-                        arcname=".",
-                    )
-
-                _LOGGER.info("Backup folder %s done", name)
-                self._data[ATTR_FOLDERS].append(name)
-            except (tarfile.TarError, OSError) as err:
-                _LOGGER.warning("Can't backup folder %s: %s", name, err)
+            _LOGGER.info("Backing up folder %s", name)
+            with SecureTarFile(
+                tar_name, "w", key=self._key, gzip=self.compressed
+            ) as tar_file:
+                atomic_contents_add(
+                    tar_file,
+                    origin_dir,
+                    excludes=[],
+                    arcname=".",
+                )
+
+            _LOGGER.info("Backup folder %s done", name)
+            self._data[ATTR_FOLDERS].append(name)
 
         # Save folder sequential
         # avoid issue on slow IO
         for folder in folder_list:
             try:
                 await self.sys_run_in_executor(_folder_save, folder)
-            except Exception as err:  # pylint: disable=broad-except
-                _LOGGER.warning("Can't save folder %s: %s", folder, err)
+            except (tarfile.TarError, OSError) as err:
+                raise BackupError(f"Can't backup folder {folder}", _LOGGER.error) from err
```

Comment:
```suggestion
                raise BackupError(f"Can't backup folder {folder}: {err}", _LOGGER.error) from err
```

Reply:
Hm, so we don't want a stack trace printed I guess since that is a somewhat "excepted" issue? Works for me.

Reply:
This suggestion _adds_ more info.
Use `_LOGGER.exception` if you want the stack